js/libs/core.js

```diff
@@ -883,6 +888,63 @@ Fliplet.FormBuilder = (function() {
         component.methods.openFilePicker = component.methods._openFilePicker;
       }
 
+      component.methods._getFieldOptions = function() {
+        var $vm = this;
+        var fields = $vm.$parent.fields;
+
+        if ($vm.fieldOptions.length < 1) {
+          $vm.fieldOptions = fields.map(function(field) {
+            if (field._type !== 'flButtons' && field._type !== 'flAddress') {
+              return { label: field.label, disabled: false };
+            }
+          }).filter(function(field) {
+            return field;
+          });
+        }
+      };
+
+      component.methods._updateDisabledOptions = function() {
+        var $vm = this;
+
+        const assignedValues = new Set(
+          Object.values($vm.selectedFieldOptions).filter(value => value)
+        );
+
+        $vm.fieldOptions.forEach(option => {
+          option.disabled = assignedValues.has(option.label);
+        });
+      };
+
+      component.methods._initAddressTypeahead = function() {
+        var $vm = this;
+        var countries;
+
+        fetch('https://restcountries.com/v3.1/all').then(function(response) {
```

Comment:
@armine-fliplet, what's that? Why do we create some third-party dependency here? The list of countries doesn't change that often. Especially since we use only part of the info. Please extract the needed data into a separate file, and store it inside the project.

Reply:
@dwfliplet  Are you suggesting we keep the data static, or should we get it from the API?

Reply:
@armine-fliplet to keep data static, but it can be kept in separate file, or even fetched from json file but hosted on our side